cczoo/psi/gramine/CI-Examples/psi/cpp/Makefile

```diff
@@ -15,14 +15,14 @@
 
 ################################# CONSTANTS ###################################
 
-THIS_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
-ARCH_LIBDIR ?= /lib/$(shell $(CC) -dumpmachine)
-ENTRYPOINT ?= ""
+GRAMINEDIR ?= ../../../..
+
+include $(GRAMINEDIR)/Scripts/Makefile.configs
 
 ifeq ($(DEBUG),1)
-GRAMINE_LOG_LEVEL = debug
+GRAPHENE_LOG_LEVEL = debug
```

Comment:
Is it a typo? Please use GRAMINE to replace GRAPHENE.

Reply:
I will fix it!